howso/utilities/feature_attributes/relational.py

```diff
@@ -928,25 +935,51 @@ def _infer_feature_bounds(self,  # noqa: C901
                                 mode_f = date_to_epoch(mode_value, format_dt)
                             else:
                                 mode_f = self._value_to_number(mode_value)
-                            if actual_min_value == mode_f:
-                                min_value = actual_min_value
-                            if actual_max_value == mode_f:
-                                max_value = actual_max_value
+                            if observed_min_value == mode_f:
+                                min_v = observed_min_value
+                            if observed_max_value == mode_f:
+                                max_v = observed_max_value
                 # If this is a datetime feature, convert back from epoch time
                 if format_dt is not None:
-                    min_value = epoch_to_date(min_value, format_dt, min_date_tz)
-                    max_value = epoch_to_date(max_value, format_dt, max_date_tz)
-                output = {'min': min_value, 'max': max_value, 'allow_null': allow_null}
+                    min_v = epoch_to_date(min_v, format_dt, min_date_tz)
+                    max_v = epoch_to_date(max_v, format_dt, max_date_tz)
+                output = {
+                    'min': min_v, 'max': max_v,
+                    'observed_min': observed_min_value, 'observed_max': observed_max_value,
+                    'allow_null': allow_null
+                }
             else:
                 # If no min/max were found from the data, use min/max size of
                 # the data type.
-                min_value, max_value = self._get_min_max_number_size_bounds(
+                min_v, max_v = self._get_min_max_number_size_bounds(
                     feature_attributes, feature_name)
-                if min_value is not None and max_value is not None:
-                    output = {'min': min_value, 'max': max_value}
+                if min_v is not None and max_v is not None:
+                    output = {
+                        'min': min_v, 'max': max_v,
+                        'observed_min': observed_min_value, 'observed_max': observed_max_value,
```

Comment:
Same as other comment, at least one of these are NaN/Null in this case.

Reply:
Fair point. Will update.